packages/assets-controllers/src/Standards/ERC20Standard.ts

```diff
@@ -55,16 +56,10 @@ export class ERC20Standard {
   async getTokenSymbol(address: string): Promise<string> {
     // Signature for calling `symbol()`
     const payload = { to: address, data: '0x95d89b41' };
-    const result = await this.provider.call(payload);
-
-    const abiCoder = new AbiCoder();
-
+    const result = (await this.provider.call(payload)) as `0x${string}`;
```

Comment:
Is this guaranteed to start with `0x`? We have some assertion utils in `@metamask/utils` which will change the type of it.

```suggestion
    const result = await this.provider.call(payload));
    assertIsStrictHexString(result);
```

Reply:
If not we have `add0x` too

Reply:
Looks like it is guaranteed to be 0x prefixed though: https://github.com/ethers-io/ethers.js/blob/master/packages/providers/src.ts/base-provider.ts#L1712

Reply:
I'm wondering if we should use an `assert` just in case. We may be assuming that `ethers` is doing the right thing today, but what if that changes in the future.

Reply:
SGTM